test/index.js

```diff
@@ -802,16 +868,22 @@ describe('Loveboat', () => {
                 let called = false;
 
                 srv.routeTransforms({
-                    name: 'post-to-patch',
-                    root: 'method',
-                    match: Joi.any().valid('post'),
-                    handler: (root, route, theServer) => {
-
-                        called = true;
-                        expect(root).to.equal('post');
-                        expect(route).to.equal(writtenRoute);
-                        expect(theServer).to.equal(srv);
-                        return 'patch';
+                    transform: {
+                        name: 'post-to-patch',
+                        root: 'method',
+                        match: Joi.any().valid('post'),
+                        handler: (root, route, theServer, transOptions) => {
+
+                            called = true;
+                            expect(root).to.equal('post');
+                            expect(route).to.equal(writtenRoute);
+                            expect(theServer).to.equal(srv);
+                            expect(transOptions.pizza).to.equal('delicious');
```

Comment:
Can we ensure that the objects themselves are the same, similarly to `expect(route).to.equal(writtenRoute)` above?